lib/debug/repl.cpp

```diff
@@ -148,7 +126,29 @@ std::optional<std::unique_ptr<Repl>> Repl::fromFile(const std::string &path)
           core::LogLevel::kFatal, "Invalid module id %d in configuration file", module_id);
         return std::nullopt;
       }
-      repl->addPwmCommands(static_cast<io::PwmModule>(module_id));
+      repl->addPwmCommands(module_id);
+    }
+  }
+
```

Comment:
...

Reply:
I think this is fine to increase readability considering the code for each command looks very similar.

Reply:
if you are concerned about readability then this should go in a separate function

i think using whitespace to fix shortcomings of your code structure is an anti-pattern

Reply:
I've removed the whitespaces, I think making a function for it does sound like a good idea though